manager/apis/app/v1alpha1/m4dmodule_types.go

```diff
@@ -142,13 +142,23 @@ type M4DModuleSpec struct {
 
 	// Reference to a Helm chart that allows deployment of the resources required for this module
 	// +required
-	Chart string `json:"chart"`
+	Chart ChartSpec `json:"chart"`
 
 	// StatusIndicators allow to check status of a non-standard resource that can not be computed by helm/kstatus
 	// +optional
 	StatusIndicators []ResourceStatusIndicator `json:"statusIndicators,omitempty"`
 }
 
+type ChartSpec struct {
+	// Name of helm chart
+	// +required
+	Name string `json:"chart"`
+
+	// Values to pass to helm chart insallation
+	// +optional
+	Values map[string]string `json:"values,omitempty"`
```

Comment:
This only allows passing fully qualified values, which is different from how Helm values are specified in a YAML files (i.e., a nested object). 

Instead, you can try to create something like `type Values map[string]interface{}` and implement DeepCopy for it as described in https://github.com/operator-framework/operator-sdk/issues/1795#issuecomment-520287739. Just need to disable auto generation of DeepCopy if it conflicts or something.

Reply:
ok, will do

Reply:
@hunchback If this becomes too hard to implement then you can skip for now. We can embed a yaml inside as a string (like others do in configmaps) or use the fully qualified names.

Reply:
ok, reverted to `map[string]string`